docs/src/examples/components/Icon/Variations/IconExampleColor.shorthand.tsx

```diff
@@ -53,6 +54,23 @@ const IconExampleColor = () => (
         variables={{ color: 'cornflowerblue', borderColor: 'orangered' }}
       />
     </div>
+    <Text
+      content={
+        <span>
+          USING THE <code>color</code> PROP:
+        </span>
+      }
+      weight="bold"
+    />
```

Comment:
```jsx
<Text weight="bold">
  USING THE <code>color</code> PROP:
</Text>
```

Will be more compact

Reply:
agreed, changed in the whole example, thx